Main Model/household-procedures.nls

```diff
@@ -22,15 +22,15 @@ to search-delivery-capable-vendor
 end
```

Comment:
I'm not sure about search-delivery-capable-vendor for household either. Let's look at how Delli Gatti households choosing firms and then decide who we'll do it.